projects/core/src/lib/colors/color-map-selector/color-map-selector.component.html

```diff
@@ -72,8 +72,8 @@
             </table>
         </div>
 
-        <mat-action-row>
+        <!-- <mat-action-row>
```

Comment:
Please add a TODO here that this will be instantiated to populate the color table

Reply:
Replaced the commented out section with a TODO f2c0f44af28f79d4d5bb91812260706d2c463892 (Did I understand this right?)

Reply:
After all. :) I put the commented out method back in for future reference df5f7fdfc5f3388cdcb12b1a05092ac86301c885